From 44aecf40489d87edaa231b6ea945255275c84ceb Mon Sep 17 00:00:00 2001
From: Cedric Poignet <cedric.poignet@intel.com>
Date: Wed, 27 Mar 2013 12:12:13 +0100
Subject: [PORT FROM MAIN][BT] Use HID USB bus to avoid BlueZ Kernel dependency

BZ: 96190

Using BUS_USB instead of BUS_BLUETOOTH allows us to get rid
of the driver registration for the BUS_BLUETOOTH made in the
HIDP kernel layer.

This modification will be part of the next AOSP.

Change-Id: Ifd1fbc4b60fd857c9bb1859287aedb65ad8ea699
Orig-Change-Id: I684708b1960c0f1a1e07038207f2a4fe138b7275
Feature: BT
Category: Integration
Purpose: ABSP-Adapter
Origin: 3rdVendors
Upstream-candidate: no
Signed-off-by: Cedric Poignet <cedric.poignet@intel.com>
Reviewed-on: http://android.intel.com:8080/99669
Reviewed-by: cactus <cactus@intel.com>
Reviewed-by: Bondier, CedricX <cedricx.bondier@intel.com>
Tested-by: Champciaux, Nicolas <nicolas.champciaux@intel.com>
Reviewed-by: buildbot <buildbot@intel.com>
Tested-by: buildbot <buildbot@intel.com>
---
 btif/co/bta_hh_co.c |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/btif/co/bta_hh_co.c b/btif/co/bta_hh_co.c
index e7f9be2..dd59c7d 100644
--- a/btif/co/bta_hh_co.c
+++ b/btif/co/bta_hh_co.c
@@ -420,7 +420,7 @@ void bta_hh_co_send_hid_info(btif_hh_device_t *p_dev, char *dev_name, UINT16 ven
     strncpy((char*)ev.u.create.name, dev_name, sizeof(ev.u.create.name) - 1);
     ev.u.create.rd_size = dscp_len;
     ev.u.create.rd_data = p_dscp;
-    ev.u.create.bus = BUS_BLUETOOTH;
+    ev.u.create.bus = BUS_USB;
     ev.u.create.vendor = vendor_id;
     ev.u.create.product = product_id;
     ev.u.create.version = version;
-- 
1.7.4.1

